README.md

```diff
@@ -8,6 +8,21 @@ This repository contains the **source code** of the official app registry for th
  </a>
 </p>
 
+## Requirements
```

Comment:
Since these requirements are related to "adding an app to the registry", it should probably be a sub-section of the "Adding  an app to the registry" section. Alternatively, you could rephrase this a bit more generally:

> All apps that are part of this registry, must adhere to the following requirements: [...]

And so on.